krita: 4.4.2 -> 4.4.3

eduardosm opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

eduardosm committed:
krita: 4.4.2 -> 4.4.3

eduardosm commented:
Result of `nixpkgs-review pr 120706` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>gmic-qt-krita (gmic_krita_qt)</li>
    <li>krita</li>
  </ul>
</details>

r-rmcgibbo commented:
Result of `nixpkgs-review pr 120706` at 111261ca run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package skipped due to time constraints:</summary>
  <ul>
    <li>gmic-qt-krita</li>
  </ul>
</details>
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>krita</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/applications/graphics/krita/default.nix:51:5:
```
   |
51 |     license = licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

eduardosm committed:
krita: fix license

Source code's readme says

> Krita as a whole is licensed under the GNU Public License, Version 3. Individual files may have a different, but compatible license.

eduardosm commented:
>     * `gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Fixed
